external/dir.proj

```diff
@@ -0,0 +1,11 @@
+<Project ToolsVersion="14.0" DefaultTargets="BuildAndTest" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="../dir.props" />
+
+  <!-- Build for all configurations -->
+  <ItemGroup>
+    <Project Include="coreclr/coreclr.depproj" />
```

Comment:
maybe it was just a temporary artifact of my changes which is no longer a problem after I refactored the traversal again.